site/search/SearchPanel.tsx

```diff
@@ -429,11 +430,7 @@ export class InstantSearchContainer extends React.Component {
 
     constructor(props: Record<string, never>) {
         super(props)
-        this.searchClient = algoliasearch(ALGOLIA_ID, ALGOLIA_SEARCH_KEY, {
-            queryParameters: {
-                clickAnalytics: "true",
-            },
-        })
```

Comment:
This wasn't doing anything, as event tracking is determined by `hasClickAnalyticsConsent` referenced elsewhere in this file.

Reply:
You're right; I checked this and indeed, a `queryID` is returned by Algolia as expected.